package.json

```diff
@@ -17,19 +21,27 @@
     }
   ],
   "devDependencies": {
-    "jshint": "0.9.1"
+    "tap-spec": "^5.0.0",
+    "tape": "^4.6.3"
   },
   "engines": {
-    "node": ">=0.6.11"
+    "node": ">=5.7.0"
   },
   "homepage": "http://github.com/Wizcorp/node-graylog2",
   "license": "MIT",
-  "main": "./graylog.js",
+  "main": "./index.js",
   "repository": {
     "type": "git",
     "url": "http://github.com/Wizcorp/node-graylog2.git"
   },
   "scripts": {
-    "test": "node ./test"
+    "test": "tape ./test/*.js | tap-spec",
+    "cover": "nyc tape ./test/*.js",
+    "lint": "jshint ./lib",
+    "bench": "node ./bench.js"
+  },
+  "dependencies": {
+    "jshint": "^2.9.5",
+    "nyc": "^12.0.2"
```

Comment:
These are dev deps :)

Reply:
Yes they are, I wonder what they are doing there. 
I will fix that and try again!